src/status_im/contexts/wallet/sheets/select_account/view.cljs

```diff
@@ -2,11 +2,14 @@
   (:require [quo.core :as quo]
             quo.theme
             [react-native.gesture :as gesture]
-            [status-im.contexts.wallet.common.utils :as wallet.utils]
             [status-im.contexts.wallet.sheets.select-account.style :as style]
             [utils.i18n :as i18n]
+            [utils.money :as money]
+            [utils.number :as number]
             [utils.re-frame :as rf]))
 
+(def ^:private rounding-decimals 4)
```

Comment:
should this be in `constants`, or is it specific to this screen only?

Reply:
This is specific for this component, as everywhere else we use 6 decimals